### PR TITLE
[4.0] Accessing property without checking the object state causing Joomla installer non-responsive

### PR DESCRIPTION
**Pull request by ghost**
After installing Joomla, go to customization and select language say Hindi for installation. If the language cannot be installed, the server will not return any value, and the value is null. But the below code is accessing its properties without checking the data is null or not.

Pull Request for Issue # .

### Summary of Changes
Added a check to see if the download URL object exists before accessing its properties.

### Testing Instructions
Install Joomla v4.0.0.alpha4
After installation complete, click on Customize installation and select Hindi language to install
You should see success message or error message.

### Expected result
Screen is not frozen

### Actual result
Error message shown and redirected to final status page

### CODE REVIEW

**Comment by wojsmol** [comment]
@corejoomla Please fix this code style error
```
[FILE: .../github.com/joomla/joomla-cms/installation/src/Model/LanguagesModel.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 257 | ERROR | Expected "if (...)\n...{...}\n...else\n"; found "if (...)
     |       | ...{...}\n...else\n"
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------]
```

File: installation/src/Model/LanguagesModel.php [hunk 1]
```diff
@@ -249,8 +249,17 @@ protected function getPackageUrl($remote_manifest)
 	{
 		$update = new Update;
 		$update->loadFromXml($remote_manifest);
+
```
Comment (Quy) [inline installation/src/Model/LanguagesModel.php]: Remove tabs

File: installation/src/Model/LanguagesModel.php [hunk 2]
```diff
@@ -249,8 +249,17 @@ protected function getPackageUrl($remote_manifest)
 	{
 		$update = new Update;
 		$update->loadFromXml($remote_manifest);
+		
+		// Get the download url from the remote manifest
+		$downloadUrl = $update->get('downloadurl', false);
+
```
Comment (Quy) [inline installation/src/Model/LanguagesModel.php]: Remove tabs

**Comment by wilsonge** [comment]
Nice find - thanks! Thanks for your first contribution to core!

**Pull request merged**